it/controllers/benefits/accommodation/QualifyingRelocationBenefitsAmountControllerISpec.scala

```diff
@@ -29,20 +33,17 @@ import utils.{EmploymentDatabaseHelper, IntegrationTest, ViewHelpers}
 
 class QualifyingRelocationBenefitsAmountControllerISpec extends IntegrationTest with ViewHelpers with EmploymentDatabaseHelper {
 
+  private val poundPrefixText = "£"
```

Comment:
also not sure how you feel about the poundPrefixText and amountInputName variables? again they are typically in every amount controller spec so would it be worth having one variable in IntegrationTest?

Reply:
fair enough